exonum-java-binding/common/src/main/java/com/exonum/binding/common/proofs/list/ListProofStructureValidator.java

```diff
@@ -77,6 +77,9 @@ public void visit(ListProofElement element) {
     listProofElementsInfo.add(new NodeInfo(element, depth));
   }
 
+  @Override
+  public void visit(ListProofOfAbsence listProofOfAbsence) {}
```

Comment:
Shan't we check that if such a node appears in the tree, it is a single node
  in the whole proof tree, with no parent nodes)?

Reply:
I thought of that and specifically checked with core - although proof of absence is always a root node, they don't validate that. We could do that though